### PR TITLE
Fail CI on invalid formatting

### PR DESCRIPTION
**Pull request by kciesielski**

### CODE REVIEW

File: .github/workflows/slack-notify.yml [hunk 1]
```diff
@@ -18,6 +18,6 @@ jobs:
       uses: slackapi/slack-github-action@v1.23.0
       with:
         channel-id: ${{ vars.SLACK_NOTIFICATIONS_CHANNEL }}
-        slack-message: "The CI/CD workflow failed, see ${{ github.event.workflow_run.html_url }}"
+        slack-message: "[${{ github.event.repository.name }}] :warning: The CI/CD workflow failed, see ${{ github.event.workflow_run.html_url }}"
```
Comment (rucek) [inline .github/workflows/slack-notify.yml:21]: **note:** This could be a separate PR, since the change is unrelated to this PR's description.

**Review by rucek — APPROVED** [review]

File: .github/workflows/slack-notify.yml (hunk 1 above)
Comment (kciesielski) [reply .github/workflows/slack-notify.yml:21]: I thought about it but wasn't sure if it was worth separating. Finally, I rationalized to myself that it's a little related, because it improves the message shown when there's incorrect formatting ;)
